_dprhtml/features/navigation/tab.html

```diff
@@ -34,7 +34,7 @@
 
   <div class="card mb-2">
     <div class="card-header" style="line-height: 1.35rem;">
-      Navigate book hierarchy
+      Navigate Book Hierarchy
```

Comment:
Welcome Aboard :)

Reply:
@cesiumetta congratulations on your first PR. Hadn't seen this yet.